.eslintrc.yml

```diff
@@ -1,2 +1,12 @@
+parser: babel-eslint
+plugins:
+  - node
+  - import
+  - promise
```

Comment:
We are already using all of these plugins in `eslint-config-backend`?

Reply:
https://github.com/enkidevs/eslint-configs/blob/master/packages/eslint-config-backend/index.js